benchmarks/NonStiffBVP/nonlinear_wpi.jmd

```diff
@@ -39,6 +39,24 @@ abstols = 1.0 ./ 10.0 .^ (1:4)
 reltols = 1.0 ./ 10.0 .^ (1:4);
 ```
 
+Function that checks retcode.
+
+```julia
+function retcode_check(setups, labels)
+    success_setups = []
+    success_labels = []
+    for (setup, label) in zip(setups, labels)
+        temp_sol = solve(prob, setup[:alg], abstol=1/10^8,reltol=1/10^8, dt=0.1)
+        test = TestSolution(temp_sol.t, temp_sol.u)
+        if SciMLBase.successful_retcode(test.retcode)
```

Comment:
What is this checking? It should be checking the recode fo the solution returned by `solve`

Reply:
Should be fixed now